estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,92 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError, ValidationError
+from datetime import timedelta
```

Comment:
```suggestion
from datetime import timedelta

from odoo import api, fields, models
from odoo.exceptions import UserError, ValidationError
```
As per our import guidelines https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#python